core/templates/pages/contributor-dashboard-page/services/contribution-and-review.service.ts

```diff
@@ -170,6 +182,171 @@ export class ContributionAndReviewService {
     );
   }
 
+  async fetchTranslationSuggestionsAsync(
+      fetcher: SuggestionFetcher,
+      shouldResetOffset: boolean,
```

Comment:
Are these parameters even needed? Since we're not fetching one page at a time anymore, then it doesn't seem like we do.

Reply:
Yes, updated the parameters.